### ISSUE TITLE
Read loop points from audio in existing nus3audio file

### ISSUE THREAD

**Issue by junetried** [junetried/simple-nus3audio-gui#2]
Currently, loop points are only written to audio. When reading a nus3audio file, all loop points are gone.

Ideally, we could read the loop points from the audio when we load it.

**Comment by junetried** [comment]
VGAudioCli has the `--metadata` option, which works for idsp files, but lopus files return an exception:
```
Unhandled Exception:
System.Collections.Generic.KeyNotFoundException: The given key 'NxOpus' was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <f9b49af01f044e55b62df81b80d829b3>:0 
  at VGAudio.Cli.Metadata.Print.PrintMetadata (VGAudio.Cli.Options options) [0x0001e] in <734b85c7de01404395eba21e14ddc4c6>:0 
  at VGAudio.Cli.Converter.RunConverterCli (System.String[] args) [0x00071] in <734b85c7de01404395eba21e14ddc4c6>:0 
  at VGAudio.Cli.VGAudioCli.Main (System.String[] args) [0x00000] in <734b85c7de01404395eba21e14ddc4c6>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.Collections.Generic.KeyNotFoundException: The given key 'NxOpus' was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <f9b49af01f044e55b62df81b80d829b3>:0 
  at VGAudio.Cli.Metadata.Print.PrintMetadata (VGAudio.Cli.Options options) [0x0001e] in <734b85c7de01404395eba21e14ddc4c6>:0 
  at VGAudio.Cli.Converter.RunConverterCli (System.String[] args) [0x00071] in <734b85c7de01404395eba21e14ddc4c6>:0 
  at VGAudio.Cli.VGAudioCli.Main (System.String[] args) [0x00000] in <734b85c7de01404395eba21e14ddc4c6>:0
```
That's a problem, especially since lopus is the format used for music, the audio most likely to be looped.

If we were willing to take on a new dependency, [vgmstream](https://github.com/vgmstream/vgmstream) is able to print metadata, and for both formats.

**Comment by junetried** [comment]
This is fixed with 31f44a56e8cae9e60fdd64f9b55a5bdf085f2803. Currently the vgmstream path is not configurable in-app - this issue can be closed once it properly is.

**Comment by junetried** [comment]
The path is configurable as of faafe9017e2c46e83835f88b5bf8205e20b96b49.
